CHANGELOG.cdf-tk.md

```diff
@@ -15,6 +15,13 @@ Changes are grouped as follows:
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## TBD
+
+### Improved
+
+- Gives a more informative error message when the authentication segment of a transformation resource file is
+  missing a required field.
```

Comment:
You have also added support for writing the SQL into the transformation yaml file, would add that also to the changelog